tests/js/component_api.spec.js

```diff
@@ -22,6 +22,22 @@ test('set component data item value', async () => {
         expect(payload.actionQueue[0].payload.name).toEqual('foo')
         expect(payload.actionQueue[0].payload.value).toEqual('bar')
     })
+
+    component.set('foo','bar')
```

Comment:
I like the PR, but I don't think this test actually excercises the change.

You would need to test for the actual HTML string being used for an event handler or something. Something like:

`<button wire:click="callSomething('foo' , 'bar')">`

Does that make sense?

Reply:
totally agree, opened a new PR due to rebase issus #551